### PR TITLE
baic front done

### PR DESCRIPTION
**Pull request by MaxTrgb**

### CODE REVIEW

**Review by BlackOrangee — APPROVED** [review]
very nice

File: my-app/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```
Comment (XardasCode) [inline my-app/README.md:1]: Can we remove this file? Or make README for our project?

File: my-app/src/Home/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+  return (
+    <footer className='myFooter'>
+      
+      {isVisible && (
+        <button onClick={scrollToTop} className="backToTopBtn">
+          Back to Top
+        </button>
+      )}
+      <p>© 2022 Denmap. All rights reserved.</p>
```
Comment (XardasCode) [inline my-app/src/Home/Footer.jsx:38]: 2022?

Comment (MaxTrgb) [reply my-app/src/Home/Footer.jsx:38]: done

File: my-app/README.md (hunk 1 above)
Comment (MaxTrgb) [reply my-app/README.md:1]: okay

**Pull request merged**